beetmoverscript/templates/firefox_nightly_en_us_signed.yml

```diff
@@ -5,20 +5,16 @@ mapping:
 {% for locale in ['en-US'] %}
   {{ locale }}:
   {% if platform in ["linux-i686", "linux-x86_64"] %}
-    package:
-      artifact: target.tar.bz2
+    target.tar.bz2:
       s3_key: firefox-{{ version }}.{{ locale }}.{{ platform }}.tar.bz2
-    complete_mar:
-      artifact: update/target.complete.mar
+    target.complete.mar:
       s3_key: firefox-{{ version }}.{{ locale }}.{{ platform }}.complete.mar
       update_balrog_manifest: true
-    checksums:
-      artifact: target.checksums
-      s3_key: firefox-{{ version }}.{{ locale }}.{{ platform }}.checksums
-    checksums_asc:
-      artifact: target.checksums.asc
-      s3_key: firefox-{{ version }}.{{ locale }}.{{ platform }}.checksums.asc
-
+# Bug 1316124 - figure out checksums
```

Comment:
I think for Firefox it's safe to uncomment these. The files are correctly attached to the signing task and are specified in the upstreamArtifacts. At least in your fake beetmover task https://queue.taskcluster.net/v1/task/YkE_OQjoTj2G92P4A4Cy1A as well as in the README. IIUC we ought to tweak the firefox/fennec in-tree tasks as well here https://bug1317800.bmoattachments.org/attachment.cgi?id=8823538 to encompass the checkums?